### ISSUE TITLE
Improve the Parallel Performance of `kgtk import-wikidata`

### ISSUE THREAD

**Issue by CraigMiloRogers** [usc-isi-i2/kgtk#120]
The current parallelization implementation for `kgtk import-wikidata` has the worker tasks open the task-specific output file each time a new task arrives, closing the file when the task is complete. This seems suboptimal.

Change the implementation, if possible, to support opening the task-specific output files only once per worker task.

Use the enter() and exit() hooks in pyrallel to implement this.

**Comment by CraigMiloRogers** [comment]
I'm testing a fix to this problem.

**Comment by CraigMiloRogers** [comment]
Committed.
